### PR TITLE
Consensus

### PR DESCRIPTION
**Pull request by ebuchman**
- fuck tex (i guess)
- some updates to consensus (address weak synchrony concern from #10 - we should cite honeybadger, and should add it to related work, where we can discuss synchrony assumptions more)
- hub = gnucleus (at least when introduced, should we change everywhere else?)
- start thinking about citation format - it may be best to adopt pandoc (document converter) and utilize their markdown syntax extensions, though not sure what it would do to readability on github (which won't really matter since pandoc should give us pdfs). alternatively, we just use markdown's link reference syntax as I did here for eg. flp citation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,20 +88,26 @@ protocol should provide additional guarantees in the event that the tolerance
...
-Tendermint is a Byzantine fault-tolerant (BFT) consensus protocol for
-asynchronous networks, notable for its simplicity, performance, and
-fork-accountability.  The protocol requires a fixed, known set of N validators,
-where the ith validator is identified by its public key, `V_i`. Validators
-attempt to come to consensus on one block at a time, where a block is a list of
-transactions.  Consensus on a block proceeds in rounds. Each round has a
+Note this is much unlike Bitcoin, where forking is a regular occurence due to network asynchrony
+and the probabilistic nature of finding partial hash collissions. 
+Since in many cases, a malicious fork is indistinguishable from a fork due to asynchrony,
+Bitcoin can not reliably implement fork-accountability.
```
Comment (jaekwon) [inline README.md]: They kind of do, by paying for the block before it's mined.

File: README.md [hunk 2]
```diff
@@ -111,31 +116,37 @@ block in the same round a Polka, and a collection of +⅔ PreCommits for a singl
...
 come with justification, in the form of a Polka for that block. If the validator
 has already PreCommit a block at round `R_1`, we say they are "locked" on that
-block, and the Polka used to justify the new PreCommit at round `R_2` must come
-in a round `R_polka` where `R_1 < R_polka <= R_2`.  Second, validators must
-Propose and/or PreVote the block they are "locked" on.  Together, these
-conditions ensure that a validator does not PreCommit without sufficient
-evidence, and that validators which have already PreCommit cannot contribute to
-evidence to PreCommit something else.  This ensures both safety and liveness of
-the consensus algorithm.
+block, and the Polka used to justify the new PreCommit at round R_2 must come in
```
Comment (jaekwon) [inline README.md]: R_2 -> `R_2`

**Comment by zmanian** [comment]
+1 for gnucleus

File: README.md [hunk 3]
```diff
@@ -111,31 +116,37 @@ block in the same round a Polka, and a collection of +⅔ PreCommits for a singl
...
 
 Tendermint’s security derives simultaneously from its use of optimal Byzantine
-fault-tolerance and the locking mechanism.  The former ensures that ⅓ or more
-validators must be Byzantine to cause a violation of safety, where more than two
-values are committed.  The latter ensures that, if ever any set of validators
-attempts, or even succeeds, in violating safety, they can be identified by the
-protocol.  This includes both voting for conflicting blocks and broadcasting
+fault-tolerance and the locking mechanism.  Together, they ensure that:
+* ⅓+ validators must be Byzantine to cause a violation of safety, where more than two
+values are committed.  The latter ensures that,
```
Comment (jaekwon) [inline README.md]: I don't think we should refer to BFT and the locking mechanism as separate things.  Together they are a BFT, and together they ensure...

File: README.md (hunk 1 above)
Comment (ebuchman) [inline README.md]: they kind of do. but forking is still a regular/expected occurrence, and its not punished explicitly ...

File: README.md (hunk 3 above)
Comment (ebuchman) [inline README.md]: agreed. that was my intention here my removing "the former/the latter" bits but should change first sentence too

**Pull request merged**